media-libs/libffado/libffado-2.3.0.ebuild

```diff
@@ -0,0 +1,116 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python2_7 )
+[[ "${PV}" = "9999" ]] && inherit subversion
+inherit eutils python-single-r1 scons-utils toolchain-funcs udev multilib-minimal
```

Comment:
Do we have to support multilib for this package?

Or as a general question: Do we want to support multilib?
I have no experience with maintaining ebuilds for multilib, but it seems quiet a bit more involved than it is without it.